Twig/Extension/SecurityExtension.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Admingenerator\GeneratorBundle\Twig\Extension;
+
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+/**
+ * @author Stéphane Escandell <stephane.escandell@gmail.com>
+ */
+class SecurityExtension extends \Twig_Extension
+{
+    /**
+     * @var AuthorizationCheckerInterface
+     */
+    private $authorizationChecker;
+
+    /**
+     * @var bool
+     */
+    private $useExpression;
+
+    /**
+     * @param AuthorizationCheckerInterface $authorizationChecker
+     * @param bool $useExpression
+     */
+    public function __construct(AuthorizationCheckerInterface $authorizationChecker, $useExpression = false)
+    {
+        $this->authorizationChecker = $authorizationChecker;
+        $this->useExpression = $useExpression;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            'is_one_admingenerator_granted' => new \Twig_SimpleFunction('is_one_admingenerator_granted', array($this, 'isOneGranted')),
+        );
+    }
+
+    /**
+     * @param  array  $credentials
+     * @return bool
+     */
+    public function isOneGranted(array $credentials, $object = null)
```

Comment:
I really like this change. Previous implementation was more cumbersome. And since this is defined as a service, it can be easily overriden/tweaked to implement custom behaviour. Great job!